pom.xml

```diff
@@ -117,6 +117,200 @@
+        <jaxb.version>2.2.11</jaxb.version>
+        <jaxb2.version>0.9.4</jaxb2.version>
+        <jboss-logging.version>3.3.1.Final</jboss-logging.version>
+        <jcodec.version>0.2.0_1</jcodec.version>
+        <jdom.bundle.version>1.1_4</jdom.bundle.version>
+        <jetty.solr.version>9.4.10.v20180503</jetty.solr.version>
+        <jetty.version>9.4.11.v20180605</jetty.version>
+        <jgroups.version>3.6.13.Final</jgroups.version>
+        <joda-convert.version>1.7</joda-convert.version>
+        <jodah-failsafe.version>0.9.5</jodah-failsafe.version>
+        <joda-time.version>2.10</joda-time.version>
+        <johnzon.version>0.9.5</johnzon.version>
+        <jpeg2000.version>1.3.1_CODICE_3</jpeg2000.version>
+        <jsonassert.version>1.5.0</jsonassert.version>
+        <json-simple.version>1.1.1</json-simple.version>
+        <json-smart.version>2.3</json-smart.version>
+        <jsoup.version>1.11.3</jsoup.version>
+        <jsr305.version>3.0.2_1</jsr305.version>
+        <jsr339-api.bundle.version>2.6.0</jsr339-api.bundle.version>
+        <jts.bundle.version>1.14.0_1</jts.bundle.version>
+        <jts.spatial4j.version>1.15.0</jts.spatial4j.version>
+        <jts.version>1.14.0</jts.version>
+        <jvnet-ogc.version>2.6.1</jvnet-ogc.version>
+        <karaf.version>4.2.1</karaf.version>
+        <keyczar.version>0.66</keyczar.version>
+        <la4j.version>0.6.0</la4j.version>
+        <log4j.version>1.2.17</log4j.version>
+        <logback.classic.version>1.2.3</logback.classic.version>
+        <logback.version>1.2.3</logback.version>
+        <lucene.version>7.4.0</lucene.version>
+        <lux.version>1.2</lux.version>
+        <!-- Mina version is not the latest version due to Apache FTPServer 1.0.6 not supporting newer releases properly -->
+        <mina.version>2.0.6</mina.version>
+        <moment.version>2.20.1</moment.version>
+        <mpegts-streamer.version>0.1.0_2</mpegts-streamer.version>
+        <net.minidev.asm.version>1.0.2</net.minidev.asm.version>
+        <netty.version>4.1.24.Final</netty.version>
+        <node.version>v8.9.4</node.version>
+        <objenesis.version>2.5.1</objenesis.version>
+        <ogc-tools-gml-jts.version>1.0.3</ogc-tools-gml-jts.version>
+        <opendj-embedded.app.version>1.3.3</opendj-embedded.app.version>
+        <opengis.bundle.version>${opengis.version}_1</opengis.bundle.version>
+        <opengis.version>19.1</opengis.version>
+        <openlayers.version>4.6.5</openlayers.version>
+        <opensaml.osgi.version>3.3.0_2</opensaml.osgi.version>
+        <org.geotools.bundle.version>${org.geotools.version}_2</org.geotools.bundle.version>
+        <org.geotools.version>19.1</org.geotools.version>
+        <!-- Note: The swissbox version needs to match the version used by pax-exam . -->
+        <org.ops4j.pax.swissbox.version>1.8.2</org.ops4j.pax.swissbox.version>
+        <org.rrd4j.version>2.2</org.rrd4j.version>
```

Comment:
❗️ This is overriding the property that was already defined at line 119

Reply:
👍